pkg/kubelet/cm/devicemanager/kubelet_internal_checkpoint

```diff
@@ -0,0 +1 @@
+{"PodDeviceEntries":[{"PodUID":"e0a6c528-decc-11e7-9e96-c85b76944bc7","ContainerName":"e0a6c5c4-decc-11e7-9e96-c85b76944bc7","ResourceName":"domain2.com/resource2","DeviceIDs":["dev3"],"AllocResp":"CgwKBGtleTISBHZhbDI="},{"PodUID":"e0a6c528-decc-11e7-9e96-c85b76944bc7","ContainerName":"e0a6c5c4-decc-11e7-9e96-c85b76944bc7","ResourceName":"domain1.com/resource1","DeviceIDs":["dev1","dev2"],"AllocResp":"CgwKBGtleTESBHZhbDESKgoVL2NvbnRhaW5lcl9kaXIxL2ZpbGUxEg9ob3N0X2RpcjEvZmlsZTEYARIqChUvY29udGFpbmVyX2RpcjEvZmlsZTISD2hvc3RfZGlyMS9maWxlMhgBGhkKCC9kZXYvYWFhEggvZGV2L2FhYRoDbXJ3GhkKCC9kZXYvYmJiEggvZGV2L2JiYhoDbXJ3GhkKCC9kZXYvY2NjEggvZGV2L2NjYxoDbXJ3"},{"PodUID":"e0a6dee8-decc-11e7-9e96-c85b76944bc7","ContainerName":"e0a6df09-decc-11e7-9e96-c85b76944bc7","ResourceName":"domain2.com/resource2","DeviceIDs":["dev4"],"AllocResp":"CgwKBGtleTISBHZhbDM="}],"RegisteredDevices":{"domain1.com/resource1":["dev1","dev2"],"domain2.com/resource2":["dev3","dev4"]}}
```

Comment:
We really need to fix the internal checkpoint not being cleaned up bug...

Reply:
remove this file?

Reply:
Removed!